common/models.py

```diff
@@ -64,14 +122,12 @@ class RegionAbstractBase(AbstractBase):
         3. sub-county,
         4. ward
     """
-
     name = models.CharField(
         max_length=100, unique=True,
         help_text="Name og the region may it be e.g Nairobi")
-    code = models.CharField(
-        max_length=100, unique=True,
-        help_text="A unique_code 4 digit number representing the region.",
-        editable=False)
+    code = models.IntegerField(
```

Comment:
The code field editable=False was forgotten. This can come back in a subsequent PR